tests/qa_automation/patch_and_reboot.pm

```diff
@@ -40,6 +40,7 @@ sub run {
     zypper_call(q{mr -d $(zypper lr | awk -F '|' '{IGNORECASE=1} /nvidia/ {print $2}')}, exitcode => [0, 3]);
     zypper_call(q{mr -e $(zypper lr | awk -F '|' '/Basesystem-Module/ {print $2}')}, exitcode => [0, 3]) if get_var('FLAVOR') =~ /TERADATA/;
 
+    add_suseconnect_product(get_addon_fullname('phub')) if check_var('PATTERNS', 'all') && is_sle('15-SP6+');
```

Comment:
@dzedro can you please add `is_sle('<16);` as well?

Reply:
Done, they don't have allpatterns or didn't find such test